coinstacks/bitcoin/api/src/controller.ts

```diff
@@ -169,6 +190,7 @@ export class Bitcoin extends Controller implements BaseAPI, BitcoinAPI {
       txid: '02cdb69a97d1b8585797ac31a1954804b40a71c380a3ede0793f21a2cdfd300a',
       value: '729',
       vout: 1,
+      path: "m/44'/0'/0'/0/2",
```

Comment:
This should ideally be 2 example responses, one for the address response (no path or address) and one for the xpub response (includes address and path)